apps/website/pages/storybook.html

```diff
@@ -0,0 +1,42 @@
+---
+title: Origami Storybook
+description: Storybook for Origami components and services
+permalink: /storybook/
+layout: o-layout-landing
+---
+
+<div class="o-layout__hero o-layout__hero--muted o-layout-typography">
+	<h1>{{page.title}}</h1>
+	<p>
+		Storybook is allowing us to make the most of a large ecosystem of tools to
+		help speed up design and development at the FT.
+	</p>
+</div>
+
+<div class="o-layout__main o-layout-typography">
+	<div class="o-layout__overview">
+		<div class="o-layout-item">
+			<p>
+				We’re writing JSX templates for our components for the first time, which
+				we may make public in the near future. For teams who also use JSX
+				templates, this could hugely reduce the complexity of implementing
+				Origami components in a project and make our most tricky kind of
+				migration (markup changes) simple when upgrading from one major version
+				of a component to another.
+			</p>
+		</div>
+		<div class="o-layout-item">
+			<h2>Supported Brands</h2>
+			<li>
+				<a href="https://origami.ft.com/storybook/brands/core/">
+					Core
+				</a>
+			</li>
+			<li>
+				<a href="https://origami.ft.com/storybook/brands/internal/">
+					 Internal
+				</a>
```

Comment:
A relative link here means they'll work in the preview build! :D 
```suggestion
				<a href="/storybook/brands/core/">
					Core
				</a>
			</li>
			<li>
				<a href="/storybook/brands/internal/">
					 Internal
				</a>
```